.github/workflows/docs.yml

```diff
@@ -11,10 +11,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - name: Setup node 😻
+        uses: actions/setup-node@v3
         with:
-          node-version: 16.x
-          cache: "yarn"
+          node-version-file: '.node-version'
```

Comment:
Can now point actions/setup-node at our `.node-version` file, instead of manually specifying.

Reply:
Didn't know this was possible, pretty cool